From d18331bd298148f44886768ac509f85fad35df7d Mon Sep 17 00:00:00 2001
From: Jakob Kastelic <kastelic.jakob@gmail.com>
Date: Fri, 19 Sep 2025 08:32:18 -0700
Subject: [PATCH 1/1] increase fip partition size

The default fip partition size is set to 2M, which is enough for U-Boot
but not for the Linux kernel. Here we increase it to 8M.

Signed-off-by: Jakob Kastelic <kastelic.jakob@gmail.com>
---
 .../stmicroelectronics/common/stm32mp1xx/genimage.cfg.template  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/stmicroelectronics/common/stm32mp1xx/genimage.cfg.template b/board/stmicroelectronics/common/stm32mp1xx/genimage.cfg.template
index c2affd98..2f5790ac 100644
--- a/board/stmicroelectronics/common/stm32mp1xx/genimage.cfg.template
+++ b/board/stmicroelectronics/common/stm32mp1xx/genimage.cfg.template
@@ -13,7 +13,7 @@ image sdcard.img {
 
 	partition fip {
 		image = "fip.bin"
-		size = 2M
+		size = 8M
 	}
 
 	partition rootfs {
-- 
2.39.5

